unity-ktwo/Assets/Scripts/Controllers/EnemyController.cs

```diff
@@ -172,15 +174,14 @@ void AttackPlayer(Collider other)
             StartCoroutine(StartAttackCooldown());
         }
 
-        if (player.currentHealth <= 0) 
+        if (player.currentHealth <= 0)
         {
             FindNewTarget();
         }
     }
 
     void AttackZombie(Collider other)
     {
-        if (!isServer) return;
         var zombie = other.gameObject.GetComponent<DamagableEnemy>();
```

Comment:
Should add back in.

Reply:
ok, added back into the places where I thought appropriate. let me know.

Reply:
looks fine to me